frontend/src/scenes/data-management/DataManagementPageTabs.tsx

```diff
@@ -32,12 +28,26 @@ const eventsTabsLogic = kea<eventsTabsLogicType>({
             },
         ],
     },
-    actionToUrl: () => ({
-        setTab: ({ tab }) => tabUrls[tab as DataManagementTab] || urls.events(),
+    selectors: () => ({
+        tabUrls: [
+            () => [featureFlagLogic.selectors.featureFlags],
+            (featureFlags) => ({
+                [DataManagementTab.EventPropertyDefinitions]: urls.eventPropertyDefinitions(),
+                [DataManagementTab.EventDefinitions]: urls.eventDefinitions(),
+                ...(featureFlags[FEATURE_FLAGS.SIMPLIFY_ACTIONS]
+                    ? {}
+                    : {
+                          [DataManagementTab.Actions]: urls.actions(),
```

Comment:
During QA, I found it quite jarring that my actions suddenly disappeared and are part of the main "events" list now even while I knew it was coming. Should we consider a way to point users towards the change? For example, maybe a temporary banner on the page for people who have the flag enabled


@clarkus

Reply:
Why did we move all the actions into events instead of renaming the tab to calculated events (or whatever)?